docs/integrations/integration-cache.md

```diff
@@ -71,9 +71,9 @@ integration_context = {
     'access_token': token,
     'valid_until': time_now + 3600  # Assuming the expiration time is 1 hour
 }
-demisto.setIntegrationContext(integration_context)
+set_integration_context(integration_context)
 ```
 
 ### For more examples, refer to following integrations:
  - [Microsoft Graph](https://github.com/demisto/content/blob/master/Packs/ApiModules/Scripts/MicrosoftApiModule/MicrosoftApiModule.py)
- - [EWS v2](https://github.com/demisto/content/blob/master/Packs/EWS/Integrations/EWSv2/EWSv2.py) 
\ No newline at end of file
+ - [EWS v2](https://github.com/demisto/content/blob/master/Packs/EWS/Integrations/EWSv2/EWSv2.py) 
```

Comment:
Both EWS v2 and Microsofts Graph use the ole function, need to change this as well

Reply:
graph uses the wrappers, so its good
ill replace ews with snow